contents/docs/tutorials/feature-flags.md

```diff
@@ -0,0 +1,206 @@
+---
+title: How to Safely Roll Out New Features
+sidebar: Docs
+showTitle: true
+---
+<br />
+
+<small class="note-block centered">_Estimated Reading Time: 8 minutes ☕☕_</small>
+
+<br />
+
+
+![](../../images/tutorials/banners/feature-flags.png)
+
+<br />
+
+Rolling out new features in your product can really be a pain.
+
+As a company that is growing fast and [consistently putting out new functionality](/blog/we-ship-whenever), we know this very well. 
+
+You might have great tests and robust CI/CD workflows in place, but there's always the chance that something will go wrong, or that your users may not appreciate the change.
+
+To make this process less painful and safer, you can use feature flags.
+
+Feature flags allow you to safely deploy and roll back new features. It means you can deploy features and then slowly roll them out to your users. If something has gone wrong, you can roll back new features without having to re-deploy your application.
+
+This tutorial will walk you through creating and using a feature flag with PostHog, cover some basic use cases, and answer some of the questions you may have when using them.
+
+<span class='centered'>...</span>
+
+_Prefer to watch a video? Check out the [Feature Flags section of our demo video](https://youtu.be/aUILrrrlu50?t=950) for a shorter overview of this functionality._
+
+## Creating Feature Flags
+
```

Comment:
Add a "pre-requisites" section above step 1 with links to the correct parts of the docs to do these things?

ie something like:

### Pre-requisites

* PostHog deployed
* posthog-js or python in use?
* You have a feature you want to deploy to a subset of your user base